src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.wasm.cs

```diff
@@ -44,6 +44,18 @@ private void UpdateTextBoxView()
 			}
 		}
 
+		partial void ValidateTextRange()
+		{
+			if (Text is { Length: > 0 } text || Text.TryGetStringByRange(SelectionStart, SelectionLength, out _))
```

Comment:
This is the only call of `TryGetStringByRange` and it doesn't use the `out` parameter. I think the extension does more than what it needs to do. The logic could be much simpler and can even be inlined here?

Reply:
It was Carl's suggestion https://github.com/unoplatform/uno/pull/8265#discussion_r822025400

Reply:
Looking through PR comments, it seems my suggestion is exactly what @carldebilly is expecting in this comment: https://github.com/unoplatform/uno/pull/8265#discussion_r822016197

The comment you referenced (while is about this block of code) is unrelated to the concern of the unnecessary work.